systemd/systemd.go

```diff
@@ -270,10 +270,10 @@ func EnsureAtLeast(requiredVersion int) error {
 var osutilStreamCommand = osutil.StreamCommand
 
 // jctl calls journalctl to get the JSON logs of the given services.
-var jctl = func(svcs []string, n int, follow bool) (io.ReadCloser, error) {
+var jctl = func(svcs []string, n int, follow, namespaces bool) (io.ReadCloser, error) {
 	// args will need two entries per service, plus a fixed number (give or take
 	// one) for the initial options.
-	args := make([]string, 0, 2*len(svcs)+6)        // the fixed number is 6
+	args := make([]string, 0, 2*len(svcs)+7)        // the fixed number is 6
```

Comment:
nitpick on the comment:
```suggestion
	args := make([]string, 0, 2*len(svcs)+7)  // we have at most 7 extra arguments
```

Reply:
Nice catch, thank you!